frontend/src/App.tsx

```diff
@@ -1,7 +1,27 @@
 import React from "react";
 
+import "@mantine/core/styles.css";
+
+import { ColorSchemeScript, MantineProvider } from "@mantine/core";
+
+import { theme } from "./theme";
+
+import "@mantine/core/styles.css";
```

Comment:
we should introduce an import sorter to order these by external, internal (by level) and directive (side effect stuff like css)

i.e. 

```ts
import React from "react";
import { ColorSchemeScript, MantineProvider } form "@mantine/core";

import { theme } from "./theme";

import "@mantine/core/styles.css";
```